db/migrate/20200411161613_add_order_default_status.rb

```diff
@@ -0,0 +1,5 @@
+class AddOrderDefaultStatus < ActiveRecord::Migration[5.1]
+  def change
+    change_column_default :orders, :status, "Pending"
```

Comment:
Added to my refactor list that this default status should use the enum value instead of the string for consistency.